src/nile/core/account.py

```diff
@@ -20,8 +21,16 @@ class Account:
 
     def __init__(self, signer, network):
         """Get or deploy an Account contract for the given private key."""
-        self.signer = Signer(int(os.environ[signer]))
-        self.network = network
+        try:
+            self.signer = Signer(int(os.environ[signer]))
+            self.network = network
+        except KeyError:
+            logging.error(
+                f"\n❌ Cannot find {signer} in env."
+                "\nCheck spelling and that it exists."
+                "\nTry moving the .env to the directory outside of your project."
```

Comment:
where does this specific recommendation come from?

Reply:
When I first started using Nile, this happened to me. And I recently walked a friend through using Nile, and he also had his .env inside the project. I think it might be a common-enough problem that we can anticipate

Reply:
I always have it inside the project, never had any issues. What kind of issue is that?

Reply:
It's the `KeyError` issue (i.e. not found)...second thought, `project` might not be the correct word there^. When I say `project`, I mean the local Nile repo. Maybe `Try moving the .env to the root of your project` is a better recommendation

Reply:
please do!